src/libmain/common-args.hh

```diff
@@ -49,4 +50,21 @@ struct MixJSON : virtual Args
     }
 };
 
+struct MixRepair : virtual Args
+{
+    RepairFlag repair = NoRepair;
```

Comment:
What's the reasoning for this default? Arguably fixing things automatically is more user friendly.

Is it more expensive, or are we just cautious by default?

Reply:
Yes it's more expensive. With repair, `LocalStore::addToStore` rewrites files even if they're already valid.